mock-apns/README.md

```diff
@@ -0,0 +1,28 @@
+# APNs Mock Server
+
+## Setup
+Before you run the mock server you need to create a ssl certificate
+```bash
+openssl req -new -x509 -keyout private -out private -nodes
+```
+
+Converse it to DER format
+```bash
+openssl x509 -in private -out cert.crt -outform DER
+```
+
+Go to your JVM home directory and import the ssl certificate into the trusted certificates list:
+```bash
+cd $JAVA_HOME/jre/lib/security
```

Comment:
I think it worths mention that on mac you should run `/usr/libexec/java_home` to have the value of JAVA_HOME dir